api/api-service.js

```diff
@@ -9,20 +9,19 @@ var Seneca = require('seneca')
 
 var server = new Hapi.Server()
 
-server.connection({ 
+server.connection({
   port: PORT
 })
 
 server.register({
-  register: Chairo, 
+  register: Chairo,
   options:{
     seneca: Seneca({
       tag: 'api',
       log: 'silent',
-      legacy: { logging: false },
-      internal: { logger: require('seneca-demo-logger') },
       debug: {short_logs:true}
     })
+    .use('demo-logger')
```

Comment:
I was not aware of this difference, is it a temporary hack?
Anyway, the format isn't mixed because of log:'silent'. 
We start receiving logs after demo-logger has been initiated because it ignores the log level.
(I think this is going to change since @mcdonnelldean today mentioned on the slack channel that the log filtering is going to be moved into seneca)